README.md

```diff
@@ -308,6 +309,64 @@ az role assignment create --scope $He_ACR_Id --role acrpull --assignee $(eval $H
+az keyvault secret set -o tsv --query name --vault-name $He_Name --name "LogAnalyticsSecondaryKey" --value $(az monitor log-analytics workspace get-shared-keys -g $He_App_RG -n $He_Name --query secondarySharedKey -o tsv)
+
+# get the values
+export He_LogAnalytics_Primary_Key=$(az keyvault secret show -o tsv --query value --vault-name $He_Name --name LogAnalyticsPrimaryKey)
+export He_LogAnalytics_Secondary_Key=$(az keyvault secret show -o tsv --query value --vault-name $He_Name --name LogAnalyticsSecondaryKey)
+
+# save the environment variables
+./saveenv.sh -y
+
+
+# create Azure Container Instance running webv
+az container create -g $He_App_RG --image retaildevcrew/webvalidate:beta -o tsv --query name \
+-n ${He_Name}-webv-${He_Location} -l $He_Location \
+--log-analytics-workspace $He_LogAnalyticsId --log-analytics-workspace-key $He_LogAnalytics_Primary_Key \
+ --command-line "dotnet ../webvalidate.dll --tag $He_Location -l 1000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --summary-minutes 5"
+
+# create in additional regions (optional)
+az container create -g $He_App_RG --image retaildevcrew/webvalidate:beta -o tsv --query name \
+-n ${He_Name}-webv-eastus2 -l eastus2 \
+--log-analytics-workspace $He_LogAnalyticsId --log-analytics-workspace-key $He_LogAnalytics_Primary_Key \
+ --command-line "dotnet ../webvalidate.dll --tag eastus2 -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --summary-minutes 5"
+
+ az container create -g $He_App_RG --image retaildevcrew/webvalidate:beta -o tsv --query name \
+-n ${He_Name}-webv-westeurope -l westeurope \
+--log-analytics-workspace $He_LogAnalyticsId --log-analytics-workspace-key $He_LogAnalytics_Primary_Key \
+ --command-line "dotnet ../webvalidate.dll --tag westeurope -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --summary-minutes 5"
+
+ az container create -g $He_App_RG --image retaildevcrew/webvalidate:beta -o tsv --query name \
+-n ${He_Name}-webv-southeastasia -l southeastasia \
+--log-analytics-workspace $He_LogAnalyticsId --log-analytics-workspace-key $He_LogAnalytics_Primary_Key \
+ --command-line "dotnet ../webvalidate.dll --tag southeastasia -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --summary-minutes 5"
+
+# Query logs in Log Analytics (takes several minutes after ACI creation to see logs)
+# TODO: add more query examples?
+az monitor log-analytics query -w $He_LogAnalyticsId \
```

Comment:
I've added a very simple query. Should I add some better examples such as filtering by location and such? Or should we let the user play around with the querying?

Reply:
this is good for now - leave the TODO

We probably want to make this a separate section / file with several examples and an explanation.

Definitely want to include examples of breaking the Message (json) into properties. I will update webv tonight so that the :beta tag logs in json (add the --json-log param to the creation and the :beta tag to the image (retaildevcrew/webvalidate:beta) also remove --summary-minutes - replaced by --json-log)